tokens/README.md

```diff
@@ -125,3 +125,113 @@ recipient address. That SOL can be used to pay transaction fees on staking
 operations such as delegating stake. The rest of the allocation is put in
 a stake account. The new stake account address is output in the transaction
 log.
+
+## Distribute SPL tokens
+
+Distributing SPL Tokens works very similarly to distributing SOL, but requires
+the `--owner` parameter to sign transactions. Each recipient account must be an
+initialized SPL Token Account that shares the same mint as the sender in order
+for an allocation to succeed.
+
+Send SPL tokens to the recipients in `<RECIPIENTS_CSV>`.
+
+Example recipients.csv:
+
+```text
+recipient,amount,lockup_date
+CYRJWqiSjLitBAcRxPvWpgX3s5TvmN2SuRY3eEYypFvT,75.4,
+C56nwrDVFpPrqwGYsTgQxv1ZraTh81H14PV4RHvZe36s,10,
+7aHDubg5FBYj1SgmyBgU3ZJdtfuqYCQsJQK2pTR5JUqr,42.1,
+7qQPmVAQxEQ5djPDCtiEUrxaPf8wKtLG1m6SB1brejJ1,20,
+```
+
+You can check the status of the recipients before beginning a distribution. You must include the SPL Token mint address:
+
+```bash
+solana-tokens balances --mint <ADDRESS> --input-csv <RECIPIENTS_CSV>
```

Comment:
Using `--mint` to switch from SOL to SPL Token doesn't feel quite right, seems too sneaky.   I think `--mint` can be dropped entirely.

Each account is fetched as a part of `balances` anyway so if the account is a token account then display it's token balance, otherwise the SOL balance.   This means we don't get errors like "Token mint mismatch" or "Not a token account" but I think that's ok

Reply:
Won't that be confusing?
Also, I guess that means we need to pull each different mint in order to display the decimals for each account correctly.

Reply:
Yes you're right.  Hmmm, not sure what else we could do here alternatively.  Just that `--mint` seems a little off

Reply:
It could be a separate `spl-token-balances` subcommand?

Reply:
That seems better to me